how2meet/schemas.py

```diff
@@ -0,0 +1,31 @@
+from datetime import date, datetime
+from pydantic import BaseModel
+
+
+class Event(BaseModel):
+    id: int
+    name: str
+    date: date
+    start_time: datetime
+    end_time: datetime
+    all_day: bool
+    locatiom: str
+    host_name:str
```

Comment:
```suggestion
    organizer_name:str
```
 might be prudent to standardize on "organizer" over "host" to avoid confusion. host_name/host_password are too legit in the computing world lol

Reply:
"admin" might also work